src/OSPSuite.Core/Domain/SensitivityAnalyses/SensitivityAnalysisRunResult.cs

```diff
@@ -28,25 +28,46 @@ public IEnumerable<PKParameterSensitivity> AllPKParameterSensitivitiesFor(string
             x => string.Equals(x.QuantityPath, outputPath) && string.Equals(x.PKParameterName, pkParameterName));
       }
 
-      public IEnumerable<OutputParameterSensitivity> AllOutputParameterSensitivitiesFor(string outputPath, string parameterPath)
+      public IEnumerable<OutputParameterSensitivity> AllOutputParameterSensitivitiesBySensitivityParameterName(string outputPath, string sensitivityParameterName)
```

Comment:
@PavelBal @Yuri05 I am renaming this for clarification
Two way. By SenstitivyPArameteRName or by ParameterPath